libcxx/test/libcxx/containers/sequences/vector/abi.compile.pass.cpp

```diff
@@ -46,7 +46,33 @@ class small_iter_allocator {
   friend bool operator!=(small_iter_allocator, small_iter_allocator) { return false; }
```

Comment:
This test will now fail with the frozen C++03 headers because the frozen headers don't contain the 19 => 20 ABI break, I guess. This applies to

```
libcxx/containers/sequences/deque/abi.compile.pass.cpp
libcxx/containers/sequences/list/abi.compile.pass.cpp
libcxx/containers/sequences/vector.bool/abi.compile.pass.cpp
libcxx/containers/sequences/vector/abi.compile.pass.cpp
```